cmd/backup/script.go

```diff
@@ -454,7 +471,7 @@ func (s *script) createArchive() error {
 		return fmt.Errorf("createArchive: error walking filesystem tree: %w", err)
 	}
 
-	if err := createArchive(filesEligibleForBackup, backupSources, tarFile); err != nil {
+	if err := createArchive(filesEligibleForBackup, backupSources, tarFile, compressionAlgo); err != nil {
```

Comment:
No real need for the intermediate var.
```suggestion
	if err := createArchive(filesEligibleForBackup, backupSources, tarFile, s.c.BackupCompression); err != nil {
```